cmd/orchestration/app/controller_constructor.go

```diff
@@ -30,9 +29,9 @@ import (
 )
 
 type ControllerConstructor struct {
-	FlagConfigFile      string
-	GetLegacyConfigFunc func(voyager.Location) *legacy.Config
```

Comment:
The real problem, I think, is that autowiring functions at the moment are not configurable (they are all hardcoded), and the only way to pass configuration is via context and orchestration controller constructor

Reply:
What about moving the plugins out of wiring/registry? We can pass them as parameters into the controller constructor.